vtex-apps/tiktok-tbp
# Feature/brpa 711 call tiktok sdk

#### What is the purpose of this pull request?
- Load the TikTok SDK into an iframe
- Render the TikTok Splash Page after submitting the form and retrieving the `externalData` successfully.
- Refactor the TikTokProvider and create a few hooks to organize the connection state

#### What problem is this solving?
- This PR allows the user to fill out the startup form and proceed to the TikTok integration.

#### How to test it?
In [this workspace](https://tiktok--sandboxbrdevbrl.myvtex.com/admin/tiktok). Fill out the form and click submit. It should show the TikTok Splash Page.

#### Screenshots or example usage
![tiktok_splash](https://user-images.githubusercontent.com/1629129/152064253-f8be0acb-b289-4aab-9e6e-d2e3320a1816.gif)

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/CuMiNoTRz2bYc/giphy-downsized.gif)

#### Types of changes
